application-admintools-default/src/main/resources/templates/downloadTemplate.vm

```diff
@@ -0,0 +1,139 @@
+## ---------------------------------------------------------------------------
+## See the NOTICE file distributed with this work for additional
+## information regarding copyright ownership.
+##
+## This is free software; you can redistribute it and/or modify it
+## under the terms of the GNU Lesser General Public License as
+## published by the Free Software Foundation; either version 2.1 of
+## the License, or (at your option) any later version.
+##
+## This software is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## Lesser General Public License for more details.
+##
+## You should have received a copy of the GNU Lesser General Public
+## License along with this software; if not, write to the Free
+## Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+## 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+## ---------------------------------------------------------------------------
+
+
+
+{{html clean='false'}}
+
+#if ($found == 'true')
+  <div class="adminToolsDashboardItem">
+    <h2>$services.icon.renderHTML('download')
+      $escapetool.xml($services.localization.render('adminTools.dashboard.section.download.title'))</h2>
+    <p>$escapetool.xml($services.localization.render('adminTools.dashboard.section.download.description'))</p>
+    <hr/>
+    <ul>
+      <li>
+        <a href="#downloadFilesModal"  data-toggle="modal" data-target="#downloadFilesModal">
+          $escapetool.xml($services.localization.render('adminTools.dashboard.section.download.modal.open'))</a>
+      </li>
```

Comment:
The `download all` button should be part of the External links section, as discussed in the last meeting. But you can leave it here until https://github.com/xwikisas/application-admintools/issues/14 is implemented. Be sure to add a comment there after merging this fix